src/main/java/io/vertx/cassandra/CassandraClient.java

```diff
@@ -27,19 +27,70 @@
 import io.vertx.core.Vertx;
 
 import java.util.List;
+import java.util.UUID;
 
 /**
  * Eclipse Vert.x Cassandra client.
  */
 @VertxGen
 public interface CassandraClient {
 
-  static CassandraClient create(Vertx vertx) {
-    return create(vertx, new CassandraClientOptions());
+  /**
+   * The name of the default pool
+   */
+  String DEFAULT_POOL_NAME = "DEFAULT_POOL";
+
+
+  /**
+   * Like {@link CassandraClient#createNonShared(Vertx, CassandraClientOptions)}  but with default client options
+   */
+  static CassandraClient createNonShared(Vertx vertx) {
+    return createNonShared(vertx, new CassandraClientOptions());
+  }
+
+  /**
+   * Create a Cassandra client which maintains its own data source.
```

Comment:
Can you add a warning in the Javadoc (and the doc) about non shared client being discouraged by Datastax?

Reply:
A Vert.x Cassandra client is backed by a driver `Session`.

A shared client shared the session instance. A non shared client gets its own. But Datastax recommends to create a single session instance per application

Reply:
Got it. Done